website/scripts/website-start.sh

```diff
@@ -22,4 +29,4 @@ if [ "$should_pull" = true ]; then
 fi
 
 # Run the dev-portal content-repo start script
-REPO=$PRODUCT PREVIEW_DIR="$PREVIEW_DIR" npm run start:local-preview
```

Comment:
💭 `PREVIEW_DIR` was never an env var that the `dev-portal` website shell referenced